lib/trace-agent.js

```diff
@@ -209,7 +212,7 @@ TraceAgent.prototype.parseContextFromHeader = function(str) {
   return {
     traceId: matches[1],
     spanId: Number(matches[2]),
-    options: matches[3]
+    options: Number(matches[3])
```

Comment:
Is there any public documentation (e.g. protobuf) that you can link to that describes the schema?